src/DIRAC/RequestManagementSystem/Service/ReqManagerHandler.py

```diff
@@ -340,6 +340,16 @@ def export_getRequestStatus(cls, requestID):
             gLogger.error(f"getRequestStatus: {status['Message']}")
         return status
 
+    types_getBulkRequestStatus = [list]
+
+    @classmethod
+    def export_getBulkRequestStatus(cls, requestIDs):
+        """get requests statuses given their ids"""
+        res = cls.__requestDB.getBulkRequestStatus(requestIDs)
+        if not res["OK"]:
+            gLogger.error(f"getRequestStatus: {res['Message']}")
```

Comment:
```suggestion
            gLogger.error("getBulkRequestStatus:", res['Message'])
```
Split static and dynamic parts for logging? (and correct function string)